collector.go

```diff
@@ -59,6 +59,24 @@ var (
 		"Atmospheric pressure measurement in mb",
 		varLabels,
 		nil)
+
+  windDesc = prometheus.NewDesc(
+    prefix+"wind_kmh",
```

Comment:
I took what's coming back from the netatmo api, I don't think we should do unit conversions - what do you think?